Service/ServiceNowSubmissionRenderer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace EMS\SubmissionBundle\Service;
+
+use EMS\FormBundle\FormConfig\FormConfig;
+use EMS\FormBundle\FormConfig\SubmissionConfig;
+use EMS\FormBundle\Submit\AbstractResponse;
+use EMS\SubmissionBundle\Submit\RenderedSubmission;
+use EMS\SubmissionBundle\Transformers\ServiceNowFormDataTransformer;
+use Symfony\Component\Form\FormInterface;
+
+class ServiceNowSubmissionRenderer extends SubmissionRenderer
+{
+    public function render(SubmissionConfig $submission, FormInterface $form, FormConfig $config, AbstractResponse $response = null)
+    {
+        $transformer = new ServiceNowFormDataTransformer($form);
+        $formData = $transformer->transform();
+
+        $endpointTemplate = $this->templating->createTemplate($submission->getEndpoint());
```

Comment:
Line 19 to 25 is duplicate code from the SubmissionRenderer. This should be replaced by a call to the parent render function.

Reply:
I am wondering if we could add the SnowFormDataTransformer to the submission config. That way we do not need this class and this will get called for every future Renderer automatically.